src/plugins/openmeteo/datasource.py

```
@@ -0,0 +1,333 @@
+                unit=unit,
+                is_active=True,
+                metadata={
+                    'parameter_code': param_code,
+                    'data_source': 'openmeteo'
+                }
+            )
+            sensors.append(sensor)
+            
+        return sensors
+        
+    async def get_measurements(
```

Ruff (0.12.2)

`get_measurements` is too complex (11 > 10)

(C901)